client/my-sites/preview/main.jsx

```diff
@@ -29,6 +29,11 @@ import {
 import WebPreview from 'components/web-preview';
 import { recordTracksEvent } from 'state/analytics/actions';
 
+/**
+ * Internal dependencies
+ */
+import './style.scss';
```

Comment:
Do you need to import the `Main` styles here, if this component already imports `components/main`?

Reply:
This is importing a different style file: one that contains styles specific for the `.main.preview` element. Not the one from `components/main`.